components/Dashboard/MyFeed/MyFeed.tsx

```diff
@@ -155,12 +160,25 @@ const MyFeed: React.FC<Props> = ({ currentUser }) => {
             </Button>
             <Button
               variant={ButtonVariant.Link}
+              className={`filter-action-btn ${
+                JSON.stringify(selectedLanguageFilters) ===
+                JSON.stringify(Array.from(userLanguages.values()))
+                  ? 'active'
+                  : ''
```

Comment:
Yep yep I knew that was not the most elegant piece of code I'd ever written 😂 Ok so I just improved it quite a bit I think and we can revisit this in a follow-up.

I finally just added `lodash` since they have a nice `_.isEqual()` method that performs a deep comparison between two arrays to determine if they are equivalent (also works for comparisons between objects, dates, strings, array buffers, and more). This will also cover the issue of ordering.

So I used `_.isEqual()` to create that piece of state you suggested and then cleaned up the ternary. Does that feel like a decent improvement for the time being? (I'll merge it in but can happily revisit ASAP if you don't like this solution)